_posts/2018-04-16-the-hubris-of-man.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: The Hubris of Man
+date: 2018-04-16
+categories:
+    - SmashBros,
+    - Python,
+    - Flask,
+    - SQLAlchemy,
+    - PostgreSQL,
+    - HTML,
+    - Javascript,
+    - Jeffrey'sMajorProject
```

Comment:
sonofa

Reply:
I would change this tag list, first off to be lowercase, but slim it down to:
```
- projects
- python
- flask
- html
- javascript
```